vllm/config/compilation.py

```diff
@@ -560,6 +561,13 @@ def set_splitting_ops_for_v1(self):
                 self.cudagraph_mode = CUDAGraphMode.FULL
             self.splitting_ops = []
 
+        if envs.VLLM_ALL2ALL_BACKEND == "deepep_high_throughput" and \
+                "vllm.moe_forward" not in self.splitting_ops:
+            # exclude MoE dispatch/combine from capture by ensuring
+            # piecewise splitting includes them, so communication remains
+            # outside CUDA graphs while compute can still be graphed.
+            self.splitting_ops.append("vllm.moe_forward")
```

Comment:
I just landed https://github.com/vllm-project/vllm/pull/23273 which adds a new op `vllm.moe_forward_shared` that will need to be added to this list.

Reply:
Fixed, thanks!